experimental/auth-logic/test_data/product_team_key.pub

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
What is the difference between this key and the oak_ec_public.pem key? Is not the product team Oak?

Reply:
I don't have the Oak private key you used in the past. Since this is just a test, it doesn't matter if we use the actual keys you expect to use in real life. (In fact, it may be safer this way :)

I needed to have *some* private key to test with because:
 - I needed to make a non-expired endorsement file. the one you've been using expired back in march
 - I needed to make a rekor entry that uses the non-expired endorsement file
 - I needed to use some private key to create the rekor entry.

Does that make sense?

Reply:
Ah sorry, I misunderstood. I thought I renamed the key but I guess I did not delete the old one. It's fixed now. Thanks.